README.md

```diff
@@ -1,84 +1,66 @@
-Multiple [GHC](http://haskell.org/ghc) Versions for [Travis-CI](https://travis-ci.org) [![Build](https://img.shields.io/travis/haskell-CI/haskell-ci.svg)](https://travis-ci.org/haskell-CI/haskell-ci)
+Haskell-CI: Continuous Integration for Haskell [![Hackage](https://img.shields.io/hackage/v/haskell-ci.svg)](https://hackage.haskell.org/package/haskell-ci)[![Travis Build](https://img.shields.io/travis/haskell-CI/haskell-ci.svg)](https://travis-ci.org/haskell-CI/haskell-ci)[![Build Status](https://github.com/haskell-CI/haskell-ci/workflows/Haskell-CI/badge.svg)](https://github.com/haskell-CI/haskell-ci/actions?query=workflow%3Ahaskell-ci)
 ============================================================
 
-The purpose of this document is to describe how to set up the [`.travis.yml` script](http://about.travis-ci.org/docs/user/build-configuration/) in order to build and test your [cabalized](http://www.haskell.org/cabal) Haskell package with multiple [GHC](http://haskell.org/ghc) configurations.
-At time of writing [Travis-CI](https://travis-ci.org/) has [support for building Haskell packages](http://about.travis-ci.org/docs/user/languages/haskell/) but only for a single GHC configuration (i.e. *Haskell Platform 2012.2.0.0 with GHC 7.4.1*). By following this guide, you can set up [Travis-CI](https://travis-ci.org/) jobs which have access to the following GHC versions (all compiled for *Ubuntu Linux 12.04 LTS 64-bit*):
+This project provides a script generator for continuous-integration testing of Haskell packages. It currently provides substantive support for:
 
- - GHC 6.12.3,
- - GHC 7.0.1, GHC 7.0.2, GHC 7.0.3, GHC 7.0.4,
- - GHC 7.2.1, GHC 7.2.2,
- - GHC 7.4.1, GHC 7.4.2,
- - GHC 7.6.1, GHC 7.6.2, GHC 7.6.3,
- - GHC 7.8.1, GHC 7.8.2, GHC 7.8.3, GHC 7.8.4
- - GHC 7.10.1, GHC 7.10.2, GHC 7.10.3
- - GHC 8.0.1, GHC 8.0.2
- - GHC 8.2.1, GHC 8.2.2
- - GHC 8.4.1, GHC 8.4.2, GHC 8.4.3, GHC 8.4.4
- - GHC 8.6.1, GHC 8.6.2, GHC 8.6.3, GHC 8.6.4, GHC 8.6.5
- - GHC 8.8.1, GHC 8.8.2, GHC 8.8.3, GHC-8.8.4
- - GHC 8.10.1, GHC-8.10.2, GHC-8.10.3, GHC-8.10.4
- - GHC 9.0.1
- - GHC HEAD.
```

Comment:
I think it would be useful to keep a list of the GHC versions that are supported.